ctapipe/coordinates/telescope_frame.py

```diff
@@ -90,22 +93,24 @@ def skyoffset_to_skyoffset(from_telescope_coord, to_telescope_frame):
 
 
 @frame_transform_graph.transform(DynamicMatrixTransform, AltAz, TelescopeFrame)
-def reference_to_skyoffset(reference_frame, telescope_frame):
+def altaz_to_telescope(altaz_coord, telescope_frame):
```

Comment:
Codacy noted this:
The `altaz_coord` is not actually used, this does not in fact transform coordinates, but calculates the matrix needed. Same for `telescope_to_altaz`.
And I dont see where this function is used at all in the code. Could you elaborate here?

Reply:
Ah, its used in the astropy transforms, my bad

Reply:
So then the function signature is required to look like this I assume?

Reply:
Yes it is. The only change here is to rename the function from the originally copy/pasted code of `SkyOffsetFrame` transforms in astropy.

Reply:
Coordinate transform graph functions always have the signature `(from_coord, to_frame)`